src/UI/Component/Symbol/Factory.php

```diff
@@ -62,19 +62,17 @@ public function icon(): Icon\Factory;
 	 * description:
 	 *   purpose: >
 	 *       Glyphs map a generally known concept or symbol to a specific concept in ILIAS.
-	 *       Glyphs are used when space is scarce.
+	 *       Glyphs SHOULD NOT be used with a text label. They are used when space is scarce.
 	 *   composition: >
-	 *       A glyph is a typographical character that represents
-	 *       something else. As any other typographical character, they can be
-	 *       manipulated by regular CSS. If hovered they change their background
-	 *       to indicate possible interactions.
+	 *       A glyph is a typographical character. As any other typographical character, they can be
+	 *       manipulated by regular CSS. If hovered, they MUST change either their color OR their background-color in order to indicate possible interactions.
```

Comment:
Do not state rules in the descriptions section. Please remove. If you are sure this rule is needed, move to the rules/usage section. Note that an additional rule would need a JF decision, so better remove in this PR.